scripts/mender-qemu

```diff
@@ -37,12 +37,16 @@ else
     BUILDTMP=${BUILDDIR}/tmp-glibc
 fi
 
+UBOOT_ELF=${UBOOT_ELF:="${BUILDTMP}/deploy/images/vexpress-qemu/u-boot.elf"}
+VEXPRESS_SDIMG=${VEXPRESS_SDIMG:="${BUILDTMP}/deploy/images/vexpress-qemu/${IMAGE_NAME}-vexpress-qemu.sdimg"}
+QEMU_SYSTEM_ARM=${QEMU_SYSTEM_ARM:="${BUILDTMP}/sysroots/x86_64-linux/usr/bin/qemu-system-arm"}
```

Comment:
Use `:-`, not `:=`. The latter has some surprising semantics, and although it doesn't matter in this particular case, it's better style to use the former.